include/deal.II/grid/grid_tools.h

```diff
@@ -3114,31 +3114,7 @@ namespace GridTools
   std::vector<char>
   CellDataTransferBuffer<dim,T>::pack_data () const
   {
-    // set up a buffer and then use it as the target of a compressing
-    // stream into which we serialize the current object
-    std::vector<char> buffer;
-    {
-#ifdef DEAL_II_WITH_ZLIB
-      boost::iostreams::filtering_ostream out;
-      out.push(boost::iostreams::gzip_compressor
-               (boost::iostreams::gzip_params
-                (boost::iostreams::gzip::best_compression)));
-      out.push(boost::iostreams::back_inserter(buffer));
-
-      boost::archive::binary_oarchive archive(out);
-      archive << *this;
-      out.flush();
-#else
-      std::ostringstream out;
-      boost::archive::binary_oarchive archive(out);
-      archive << *this;
-      const std::string &s = out.str();
-      buffer.reserve(s.size());
-      buffer.assign(s.begin(), s.end());
-#endif
-    }
-
-    return buffer;
+    return Utilities::pack(*this);
```

Comment:
since the function now has only one line, could you just get rid of it altogether by calling `Utilities::pack(obj)` instead of `obj.pack_data()`?

Reply:
don't we also pack and unpack `cell_id` within the `CellDataTransferBuffer`? eventually we should be always sending/receiving those in pairs.